package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Recoil - A React state management library for complex UIs",
   "main": "dist/index.js",
   "files": [
-    "dist/recoil.js"
+    "dist/index.js",
```

Comment:
There are other files in dist that should be exported; try ‘dist/**/*.js’

You can confirm the result with ‘npm pack’ and inspecting the generated tar ball